### PR TITLE
add si and biz info section to request to omit, update page width, up…

### PR DESCRIPTION
**Pull request by BrandonSharratt**
…date colouring and pull info from si biz section to completing party section

*Issue:*https://github.com/bcgov/entity/issues/22107
https://github.com/bcgov/entity/issues/22108
https://github.com/bcgov/entity/issues/22695

*Description of changes:*
Added SI Biz Info section
Per request on 22108 added 200 pixels of width to form
Removed Canada (Citizen) tooltip as thats not information we have
Fixed colour of placeholder text in reasons text area
Completing Party section now updates some text based on selections in SI Biz info area

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-210-wc1268kn.web.app

File: btr-web/btr-common-components/composables/useFetchBcros.ts [hunk 1]
```diff
@@ -3,7 +3,11 @@ export const useFetchBcros = <T>(request, opts?) => {
     const token = useBcrosKeycloak().kc.token
 
     opts = opts || {}
-    Object.assign(opts, { headers: { Authorization: `Bearer ${token}` } })
+    let headers = {}
```
[the remark this answers is not in the document]
Comment (BrandonSharratt) [reply btr-web/btr-common-components/composables/useFetchBcros.ts:6]: I tried it without and it was removing the apikey header that I was passing in in opts

File: btr-web/btr-main-app/pages/requestToOmit.vue [hunk 2]
```diff
@@ -3,12 +3,12 @@
...
 
 const omitIndividual = useOmitIndividual()
-const { completingPartyRef, omitObscureRef } = storeToRefs(omitIndividual)
+const { completingPartyRef, omitObscureRef, siBizRef } = storeToRefs(omitIndividual)
 </script>
 
 <template>
   <div class="w-full">
-    <div class="max-w-[787px]">
+    <div class="max-w-[987px]">
```
[the remark this answers is not in the document]
Comment (BrandonSharratt) [reply btr-web/btr-main-app/pages/requestToOmit.vue:11]: designs, in the most recent comment Julie asked for 200px more width

File: btr-web/btr-main-app/components/bcros/SIBizInfo.vue [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+      <p>
+        {{ $t('labels.requestOmit.siBizIntro') }}
+      </p>
+      <BcrosSection
+        :show-section-has-errors="hasErrors(['name'])"
+        :section-title="$t('placeholders.fullName')"
+        :border="false"
+        :padded-x="false"
+        :padded-y="false"
+        class="pt-9 -ml-8 pl-8"
```
[the remark this answers is not in the document]
Comment (BrandonSharratt) [reply btr-web/btr-main-app/components/bcros/SIBizInfo.vue:72]: Negative margin was needed to match designs and is in the other components as well (I agree I dislike it but removing margin left / padding left from the section component didn't achieve the design)

File: btr-web/btr-main-app/components/bcros/BusinessSearch.vue [hunk 4]
```diff
@@ -0,0 +1,96 @@
+<script setup lang="ts">
+import { useOmitIndividual } from '@/stores/omit-individual'
+
+const omitIndividual = useOmitIndividual()
+const { siBizName } = storeToRefs(omitIndividual)
+
+const ALLOWED_LEGAL_TYPES = [
+  'BC',
```
[the remark this answers is not in the document]
Comment (BrandonSharratt) [reply btr-web/btr-main-app/components/bcros/BusinessSearch.vue:8]: Added a few more business types to this now under an unknown type comment (the others in that enum have a type they are under i.e corporation/firm)

File: btr-web/btr-common-components/components/bcros/section/index.vue [hunk 5]
```diff
@@ -71,6 +76,7 @@ const props = defineProps({
   showSectionHasErrors: { type: Boolean, required: false, default: false },
   sectionTitle: { type: String, required: false, default: undefined },
   sectionTitleIcon: { type: String, required: false, default: undefined },
+  iconColor: { type: String, required: false, default: '' },
```
[the remark this answers is not in the document]
Comment (BrandonSharratt) [reply btr-web/btr-common-components/components/bcros/section/index.vue:79]: Done

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-210-wc1268kn.web.app

File: btr-web/btr-common-components/components/bcros/section/index.vue (hunk 5 above)
[the remark this answers is not in the document]
Comment (hfekete) [reply btr-web/btr-common-components/components/bcros/section/index.vue:79]: Is it updated, haha ?

**Review by hfekete — APPROVED** [review]
Good work! Just doublecheck, I dont see the name updated for the icon, if it did not update, can you do that, and you good to merge, if its updated :D good to merge!

**Pull request merged**